src/state/login/actions.js

```diff
@@ -0,0 +1,12 @@
+// actions
+export const ADD_EMAIL = 'ADD_EMAIL';
```

Comment:
I think it'd be better to create an `index.js` file at the root of `state` to map all actions. So, import these functions into `src/state/index.js` and then map the constants from there. That way, developers have a roadmap for the types of dispatch they can fire.